src/components/MyCards/MyCards.jsx

```diff
@@ -39,7 +39,7 @@ export class MyCards extends React.Component {
 
     render() {
         return (
-            <CardList cards={this.props.myCards} handleClick={this.playCard} isEnabled={this.isEnabled} cy={"my-cards"} />
+            <CardList cards={this.props.myCards} handleClick={this.playCard} playedCard={this.props.playedCard} isEnabled={this.isEnabled} cy={"my-cards"} />
```

Comment:
Can't CardList get this itself from the store?

Reply:
It can but it's not connected to it rn, figured it would be better to just pass it as props?

Reply:
I thought the point of redux was that everything just got what it needed from the store so we didn't have to pass things down as props